p9-presentation/sections/_experiments.qmd

```diff
@@ -288,16 +294,54 @@ This conforms that potential improvements could be seen from either integrating
 </div>
 
 ::: {.fragment .fade-up}
-1. Outlier removal is beneficial to some extend
+- Outlier removal is beneficial to some extend
     - Specialty approaches needs to be developed for further improvements
 :::
 
 ::: {.fragment .fade-up}
-2. Data preprocessing affects sections of the pipeline differently
-    - ICA is more sensitive
+- Rethinking the choice of models may be a better solution 
+    - Current pipeline has a lot of complexity and moving parts
+:::
+
+::: {.notes}
+**Outlier removal**
+For the outlier removal we found that depending on the model, PLS-SM or ICA, outlier removal has varying success. For PLS-SM there seems to be neglible gains from outlier removal, while ICA benefits more from it.
+Generally, however, we determine that a "one size fits all" approach to outlier detection does not provide sufficient benefits and therefore specialty approaches needs to be developed. Perhaps this involves developing approaches that cater more specifically to each oxide.
+
+**Data preprocessing**
+
+**Rethinking**
+While the current setup with PLS and ICA allows for interpretability e.g. regression plots and independent components, the MOC model as a whole loses the benefit of that because of the complexity of the pipeline. 
+This is because you are interleaving to different models that are operating in distinct ways and adjusting everything from data preprocessing to outlier removal introduces complexity with diminishing returns.
 :::
 
+## Reflections {.smaller}
+
+<div class="experiments-spacing">
+<p class="experiments-header">Taking a critical look at our project</p>
+</div>
+
 ::: {.fragment .fade-up}
-3. Rethinking the choice of models may be a better solution 
-    - Current pipeline has a lot of complexity and moving parts
+- We accomplished most of what we wanted
+    - Finished replica
+    - Conducted meaningful experiments
+:::
+
+::: {.fragment .fade-up}
+- However... We lack depth in our experiments
+    - For example:
+    - Investigate the causes of varying RMSEs with aggregated datasets for ICA
+    - Thorough investigation of ANNs and XGBoost and why exactly they perform better
```

Comment:
Also, it would be interesting to look more closely at the outlier spectra and their compositional values.